src/gwt/src/org/rstudio/core/rebind/command/CommandBundleGenerator.java

```diff
@@ -420,12 +425,20 @@ private ImageResourceInfo generateImageBundle()
       {
          String commandId = method.getName();
 
-         String key = packageName_.replace('.', '/') + "/" + commandId + ".png";
-         if (resourceNames.contains(key))
+         String key = packageName_.replace('.', '/') + "/" + commandId;
+
+         if (resourceNames.contains(key + ".png"))
          {
             writer.println("ImageResource " + commandId + "();");
             iri.addImage(commandId);
          }
+
+         if (resourceNames.contains(key + "@2x.png"))
+         {
+            writer.println("@Source(\"" + commandId + "@2x.png\")");
```

Comment:
Some versions of CMake don't like files with `@` in their names; this problem isn't obvious when you build locally but can break on the Linux build machines in the lab. We can take the change and see how it goes but my guess is that we will want to use a different naming scheme; for e.g. pdf.js (which includes retina icons) we would up going with `_2x` instead of `@2x` to overcome this issue.

https://github.com/rstudio/rstudio/blob/f9ef3c3caf056bfcc601508920279b5dfe642ac2/src/gwt/tools/sync-pdfjs#L46-L51